srcpkgs/bubbletea/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'bubbletea'
+pkgname=bubbletea
+version=0.23.2
+revision=1
+build_style=go
+build_helper=qemu
+go_import_path="github.com/charmbracelet/bubbletea"
+go_ldflags="-X main.Version=v${version}"
+short_desc="Powerful little TUI framework"
+maintainer="zenobit <zen@osowoso.xyz>"
+license="MIT"
+homepage="https://github.com/charmbracelet/bubbletea"
+changelog="https://github.com/charmbracelet/bubbletea/releases"
+distfiles="https://${go_import_path}/archive/v${version}.tar.gz"
+checksum=15be37a5ae543cbbee3343595d51e9a719bd69d7fe9b12093465cbd68968c884
+
+post_install() {
+	vlicense LICENSE
+
+	#vtargetrun ${pkgname} man > ${pkgname}.1
```

Comment:
If these aren't going to be run, they should be removed.  Why are they commented out?